WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -759,7 +759,6 @@ public boolean onOptionsItemSelected(final MenuItem item) {
             }
             if (mViewPager.getCurrentItem() > PAGE_CONTENT) {
                 if (mViewPager.getCurrentItem() == PAGE_SETTINGS) {
-                    mPost.setFeaturedImageId(mEditPostSettingsFragment.getFeaturedImageId());
```

Comment:
Maybe we need this? It can't be the same exact line, since we update the DB record, but maybe we need to fetch a fresh copy of the PostModel and update `mPost`'s featured image id here?